lib/matplotlib/colors.py

```diff
@@ -2300,6 +2300,18 @@ def __init__(self, azdeg=315, altdeg=45, hsv_min_val=0, hsv_max_val=1,
         altdeg : float, default: 45 degrees
             The altitude (0-90, degrees up from horizontal) of the light
             source.
+        hsv_min_val : number, default: 0
+            The minimum value ("v" in "hsv") that the *intensity* map can shift the
+            output image to.
+        hsv_max_val : number, default: 1
+            The maximum value ("v" in "hsv") that the *intensity* map can shift the
+            output image to.
+        hsv_min_sat : number, default: 1
+            The minimum saturation value that the *intensity* map can shift the output
+            image to.
+        hsv_max_sat : number, default: 0
+            The maximum saturation value that the *intensity* map can shift the output
+            image to.
```

Comment:
I do not know enough about HSV saturation, but it seems a bit "weird" that the min value is larger than the max value...

Reply:
It does seem odd, but the `blend_hsv` function docstring below says:
```
The default extremes are
chose so that completely shaded points are nearly black (s = 1, v = 0)
and completely illuminated points are nearly white (s = 0, v = 1).
```
So, I think this is expected although unintuitive at first glance.